toolkit/scripts/toolchain/cross/kernel_cross.sh

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+kernelVersion="5.15.48.1"
```

Comment:
Should this kernel version be fixed?  How do we keep it up to date with the rolling kernel in mariner?